### DISCUSSION TITLE
Can runtime shorter than the forcing dataset and How to initialize SCM later than the first timestamp of forcing dataset?

### DISCUSSION THREAD

**Discussion by XiaSun-Atmos** [NCAR/ccpp-scm#314] [Configuring Output]
Can `runtime` be set smaller than the forcing dataset?  For example, I have a 21-hour forcing dataset with endDate="20160811210000" and startDate="20160811000000". Can I run SCM only for 3 hours? I tried to set the `runtime=10800`, but when checking the results, SCM runs 21 hours same as the forcing dataset. 

The second question is how to initialize the SCM in the middle, for example at 20160811060000 (6 hours later than the startDate of the forcing dataset)? 

Thank you in advance for your insight!

**Comment by grantfirl** [comment]
Hi Xia,

I'm assuming that you're question is referring to using DEPHY-based input? In that case, as the code is written today, the runtime is determined from the given startdate and enddate, and whatever is in the case input namelist as 'runtime' is ignored. You could try to just change the enddate to see if it will work to run a reduced time period. I haven't tried that, however.

Initializing in the middle of a startdate/enddate period is also not coded for right now.

**Comment by XiaSun-Atmos** [comment]
Thanks Grant! I will try to change the enddate as you suggested and see how it goes.
